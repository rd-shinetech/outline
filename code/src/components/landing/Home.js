import React from 'react';
import Header from './Header';
import Section from './Section';
import Footer from './Footer';


const Home = () => {
    return (
        <div>
            
            <Header/>

            <div className="container text-muted">
                <Section/>
                <Footer/>
            </div>
            
        </div>
    );
};



export default Home;
