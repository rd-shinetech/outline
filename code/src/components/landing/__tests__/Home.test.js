import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Home from '../Home';


describe('Home.test.js', () => {

    it('renders without crashing', () => {
        const wrapper = shallow(<Home/>);

        expect(wrapper).toHaveLength(1);

        expect(wrapper.find('Header')).toHaveLength(1);
        expect(wrapper.find('Section')).toHaveLength(1);
        expect(wrapper.find('Footer')).toHaveLength(1);   

        const tree = toJson(wrapper);
        expect(tree).toMatchSnapshot();    

    });
});
