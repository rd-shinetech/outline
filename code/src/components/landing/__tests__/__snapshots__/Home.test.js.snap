exports[`Home.test.js renders without crashing 1`] = `
<div>
  <Header />
  <div
    className="container text-muted">
    <Section />
    <Footer />
  </div>
</div>
`;
